src/manifold/utils.clj

```diff
@@ -130,16 +131,13 @@
 ;;;
 
 (defmacro defprotocol+ [name & body]
-  (when-not (resolve name)
-    `(defprotocol ~name ~@body)))
+  `(potemkin.types/defprotocol+ ~name ~@body))
 
 (defmacro deftype+ [name & body]
-  (when-not (resolve name)
-    `(deftype ~name ~@body)))
+  `(potemkin.types/deftype+ ~name ~@body))
 
 (defmacro defrecord+ [name & body]
-  (when-not (resolve name)
-    `(defrecord ~name ~@body)))
+  `(potemkin.types/defrecord+ ~name ~@body))
```

Comment:
Might as well just make the existing calls use potemkin directly, if they're swappable. As macros, they have no performance penalty, but we don't need to keep these around. The `utils` ns is undocumented, so nobody should be counting on it.